### PR TITLE
LPS-129495

### PR DESCRIPTION
**Pull request by martamedio**
cc/ @julien

### CODE REVIEW

**Comment by martamedio** [comment]
ci:forward

**Review by julien — APPROVED** [review]
LGTM

File: modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/connected_applications/view.jsp [hunk 1]
```diff
@@ -28,7 +28,7 @@ int userOAuth2AuthorizationsCount = OAuth2AuthorizationServiceUtil.getUserOAuth2
 	disabled="<%= userOAuth2AuthorizationsCount == 0 %>"
 	filterDropdownItems="<%= oAuth2ConnectedApplicationsManagementToolbarDisplayContext.getFilterDropdownItems() %>"
 	itemsTotal="<%= userOAuth2AuthorizationsCount %>"
-	propsTransformer="connected_applications/js/OAuth2ConnectedApplicationsManagementToolbarPropsTransformer"
+	propsTransformer="connected_applications/js/oAuth2ConnectedApplicationsManagementToolbarPropsTransformer"
```
Comment (lipusz) [inline modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/connected_applications/view.jsp:31]: :star_struck:

Comment (lipusz) [reply modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/connected_applications/view.jsp:31]: The little things... :smirk:

**Comment by martamedio** [comment]
ci:stop

**Comment by martamedio** [comment]
ci:test:relevant

**Comment by yichenroy** [comment]
ci:forward

**Pull request closed without merging**